NEWS.md

```diff
@@ -10,11 +10,14 @@
 * provides `inventory-view-instance-set 1.0`
 * Added Variant title and Former title to alternative title default types ([MODINVSTOR-389] (https://issues.folio.org/browse/MODINVSTOR-389))
 * Extend instance contributors schema with Authority ID ([MODINVSTOR-950](https://issues.folio.org/browse/MODINVSTOR-950))
-* provides `instance-storage 9.0`
 * Require CQL query for bulk delete of instances/holdings/items ([MODINVSTOR-576](https://issues.folio.org/browse/MODINVSTOR-576), [MODINVSTOR-901](https://issues.folio.org/browse/MODINVSTOR-901))
 * provides `instance-storage 9.0`
 * provides `holdings-storage 6.0`
 * provides `item-storage 10.0`
+* Batch update with optimistic locking disabled ([MODINVSTOR-924](https://issues.folio.org/browse/MODINVSTOR-924))
+* provides `instance-storage-batch-sync-unsafe 1.0`
+* provides `holdings-storage-batch-sync-unsafe 1.0`
+* provides `item-storage-batch-sync-unsafe 1.0`
```

Comment:
Please organise all of the `provides` and `requires` statements to be at the end of the news entries for a release (as per the convention established for these modules)

Reply:
Done.